.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,17 @@
+name: deploy
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  main:
+    if: ${{ github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login to Quay
+        run: docker login -u="${{ secrets.QUAY_USER }}" -p="${{ secrets.QUAY_SECRET }}" quay.io
```

Comment:
nit: can we use the first-class `docker/login-action` action to handle this for us?

Reply:
Totally, let me add that.